src/olympia/promoted/tests/test_models.py

```diff
@@ -615,16 +615,6 @@ def _test_unique_constraint(self, fields, should_raise=False):
 
         assert PromotedAddonPromotion.objects.create(**merged) is not None
 
-    def test_multiple_promoted_groups_per_addon_raises(self):
```

Comment:
This should now test that `multiple_promoted_groups_per_addon_application_raises`

Reply:
Is that not tested by `self._test_unique_constraint({}, should_raise=True)`?

Reply:
Personally, I'd rewrite these tests - `should_raise` is only True in one case now, so the content of _test_unique_constraint should be directly in that test.  And then the other two tests, where `should_raise=False`, would be 2 explicit lines of code, e.g.
```python
PromotedAddonPromotion.objects.create(**self.required_fields)
assert PromotedAddonPromotion.objects.create(**{**self.required_fields, 'application_id': applications.ANDROID.id}) is not None
```